### ISSUE TITLE
[Android] FirebaseApp is not initialized

### ISSUE THREAD

**Issue by lynxaegon** [chemerisuk/cordova-plugin-firebase-messaging#27]
Hi, 
I cannot use the `getToken` method
```
cordova.plugins.firebase.messaging.getToken().then(function(token) {
    console.log("Got device token: ", token);
});
```
It outputs this error.
```
07-17 16:14:01.335 16934-17134/? E/ReflectiveCordovaPlugin: Uncaught exception at #getToken
                                                            java.lang.IllegalStateException: Default FirebaseApp is not initialized in this process com.example.app. Make sure to call FirebaseApp.initializeApp(Context) first.
                                                                at com.google.firebase.FirebaseApp.getInstance(SourceFile:218)
                                                                at com.google.firebase.iid.FirebaseInstanceId.getInstance(Unknown Source:0)
                                                                at by.chemerisuk.cordova.firebase.FirebaseMessagingPlugin.getToken(FirebaseMessagingPlugin.java:85)
                                                                at java.lang.reflect.Method.invoke(Native Method)
                                                                at by.chemerisuk.cordova.support.ReflectiveCordovaPlugin$1.run(ReflectiveCordovaPlugin.java:56)
                                                                at by.chemerisuk.cordova.support.ReflectiveCordovaPlugin.execute(ReflectiveCordovaPlugin.java:42)
                                                                at org.apache.cordova.CordovaPlugin.execute(CordovaPlugin.java:98)
                                                                at org.apache.cordova.PluginManager.exec(PluginManager.java:132)
                                                                at org.apache.cordova.CordovaBridge.jsExec(CordovaBridge.java:59)
                                                                at org.crosswalk.engine.XWalkExposedJsApi.exec(XWalkExposedJsApi.java:40)
                                                                at org.chromium.base.SystemMessageHandler.nativeDoRunLoopOnce(Native Method)
                                                                at org.chromium.base.SystemMessageHandler.handleMessage(SystemMessageHandler.java:39)
                                                                at android.os.Handler.dispatchMessage(Handler.java:106)
                                                                at android.os.Looper.loop(Looper.java:164)
                                                                at android.os.HandlerThread.run(HandlerThread.java:65)
```

Is there anything i'm missing here?

Thanks

**Comment by vuneotwist** [comment]
Any update on this?

**Comment by chemerisuk** [comment]
I guess the problem is the same as at https://github.com/chemerisuk/cordova-plugin-firebase-messaging/issues/20. Should be fixed at version 1.2
